react/src/components/App.js

```diff
@@ -31,19 +31,37 @@ class App extends Component {
     this.setState({ selectedUser: username, selectedCV: cvName, view: 'edit' })
   }
 
```

Comment:
Oisko tää mahdollista siirtää Api.js:ään niin saadaan kaikki api-kutsut samaan paikkaan?

Reply:
En tiia osaanko, mutta voin yrittaa. Mietin vaan kun sprint change lahestyy etta haluttiin mahdollisimman nopeesti toiminnallista matskua masteriin :)

Reply:
Tää on hyvä!

Reply:
+1 tähän näin